SPECS/grub.spec

```diff
@@ -324,6 +331,7 @@ fi
 %config(noreplace) %{_sysconfdir}/%{name}-efi.cfg
 %dir /boot/efi/EFI/%{efidir}
 %attr(0755,root,root) /boot/efi/EFI/%{efidir}/*.efi
+%attr(0755,root,root) /boot/efi/EFI/%{efibootdir}/%{grubefibootname}
```

Comment:
The directory will be created but will not belong to any RPM. You must declare the directory as handled by RPM with: `%dir /boot/efi/EFI/%{efibootdir}`